misc/deployments/test4.gno.land/config.toml

```diff
@@ -0,0 +1,247 @@
+  # and will introspect on the listener or use UPnP
+  # to figure out the address.
+  external_address = ""
+
+  # Time to wait before flushing messages out on the connection
+  flush_throttle_timeout = "100ms"
+
+  # Peer connection configuration.
+  handshake_timeout = "20s"
+  home = ""
+
+  # Address to listen for incoming connections
+  laddr = "tcp://0.0.0.0:26656" # Change me!
+
+  # Maximum number of inbound peers
+  max_num_inbound_peers = 40
+
+  # Maximum number of outbound peers to connect to, excluding persistent peers
+  max_num_outbound_peers = 40 # Advised value is 40
+
+  # Maximum size of a message packet payload, in bytes
+  max_packet_msg_payload_size = 1024
+
+  # Comma separated list of nodes to keep persistent connections to
+  persistent_peers = ""
+
+  # Set true to enable the peer-exchange reactor
+  pex = true
+
+  # Comma separated list of peer IDs to keep private (will not be gossiped to other peers)
+  private_peer_ids = ""
+
+  # Rate at which packets can be received, in bytes/second
+  recv_rate = 5120000
+
+  # Seed mode, in which node constantly crawls the network and looks for
+  # peers. If another node asks it for addresses, it responds and disconnects.
+  #
+  # Does not work if the peer-exchange reactor is disabled.
+  seed_mode = false
+
+  # Comma separated list of seed nodes to connect to
+  seeds = ""
+
+  # Rate at which packets can be sent, in bytes/second
+  send_rate = 5120000
+  test_dial_fail = false
+  test_fuzz = false
+
+  # UPNP port forwarding
+  upnp = false
+
+  [p2p.test_fuzz_config]
+    MaxDelay = "3s"
+    Mode = 0
+    ProbDropConn = 0.0
+    ProbDropRW = 0.2
+    ProbSleep = 0.0
+
+##### rpc server configuration options #####
+[rpc]
+
+  # A list of non simple headers the client is allowed to use with cross-domain requests
+  cors_allowed_headers = ["Origin", "Accept", "Content-Type", "X-Requested-With", "X-Server-Time"]
+
+  # A list of methods the client is allowed to use with cross-domain requests
+  cors_allowed_methods = ["HEAD", "GET", "POST", "OPTIONS"]
+
+  # A list of origins a cross-domain request can be executed from
+  # Default value '[]' disables cors support
+  # Use '["*"]' to allow any origin
+  cors_allowed_origins = ["*"]
+
+  # TCP or UNIX socket address for the gRPC server to listen on
+  # NOTE: This server only supports /broadcast_tx_commit
+  grpc_laddr = ""
+
+  # Maximum number of simultaneous connections.
+  # Does not include RPC (HTTP&WebSocket) connections. See max_open_connections
+  # If you want to accept a larger number than the default, make sure
+  # you increase your OS limits.
+  # 0 - unlimited.
+  # Should be < {ulimit -Sn} - {MaxNumInboundPeers} - {MaxNumOutboundPeers} - {N of wal, db and other open files}
+  # 1024 - 40 - 10 - 50 = 924 = ~900
+  grpc_max_open_connections = 900
+  home = ""
+
+  # TCP or UNIX socket address for the RPC server to listen on
+  laddr = "tcp://127.0.0.1:26657" # Change me!
```

Comment:
If we're willing to enforce initial validators(core, devx and us) to provide rpc service, we should change this value to something like `tcp://0.0.0.0:26657`

Reply:
Yup. This value can be kept for advanced cases where people rely on a reverse proxy, but let's suggest the simple version.

Reply:
Swapped:

[867a6a7](https://github.com/gnolang/gno/pull/2508/commits/867a6a772a3ecad73842c80ec1e687dd8a95e31d)